src/BinSkim.Driver/AnalysisSummaryExtractor.cs

```diff
@@ -40,7 +40,7 @@ public static AnalysisSummary ExtractAnalysisSummary(SarifLog sarifLog, AnalyzeO
                 ToolVersion = tool.Driver.Version,
                 NormalizedPath = string.Join(";", options.TargetFileSpecifiers.Select(p => System.IO.Path.GetDirectoryName(p)).Distinct()),
                 SymbolPath = options.SymbolsPath,
-                FileAnalyzed = artifacts.Count,
+                FileAnalyzed = artifacts?.Count ?? 0,
```

Comment:
>artifacts?.Count ?? 0 [](http://example.com/codeflow?start=31&length=21)

this will prevent the exception in case the SARIF does not have artifacts.

This would happen if: you don't have the --insert hashes or --Hashes option enabled but you have the telemetry rules.